yt_dlp/extractor/brilliantpala.py

```diff
@@ -14,17 +14,17 @@ class BrilliantpalaBaseIE(InfoExtractor):
 
     def _initialize_pre_login(self):
         self._HOMEPAGE = f'https://{self._DOMAIN}'
-        self._LOGIN_API = f'{self._HOMEPAGE}/login/'
+        self._LOGIN_API = f'{self._HOMEPAGE}/login/?next='
```

Comment:
Any reason for changing the URL? I think with the added query parameter, it is more prone to breakage in case they for example add another one and the order changes.

Reply:
I agree. I copied the URL from the form on the webpage and didn't think of making it more future-proof. Fixed in 542f69b67a34d517e6cfcbf4420f967ad08aac44 .

Thank you!